admin_manual/configuration_server/occ_command.rst

```diff
@@ -72,18 +94,220 @@ this example for the ``maintenance:mode`` command::
    --no-ansi             Disable ANSI output.
    --no-interaction (-n) Do not ask any interactive question.
    
+Apps Commands
+-------------
+
+The ``app`` commands list, enable, and disable apps. This lists all of your 
+installed apps, and shows whether they are enabled or disabled::
+
+ $ sudo -u www-data php occ app:list
+ 
+Enable an app::
+
+ $ sudo -u www-data php occ app:enable external
+   external enabled
+   
+``app:check-code`` checks if the app uses ownCloud's public API (``OCP``) or 
+private API (``OC_``), and then enables the app. If the app uses the private 
+API it will print a warning::
+
+  $ sudo -u www-data php occ app:check-code activity
+    [snip]
+    Analysing /var/www/owncloud/apps/activity/extension/files_sharing.php
+    0 errors
+    Analysing /var/www/owncloud/apps/activity/extension/files.php
+    0 errors
+  App is not compliant
+   
+Disable an app::
+
+ $ sudo -u www-data php occ app:disable external
+   external disabled   
+   
+Background Jobs Selector
+------------------------
+
+Select which scheduler you want to use for controlling background jobs: Ajax, 
+Webcron, or Cron. This is the same as using the **Cron** section on your Admin 
+page.
+
+This example selects Ajax::
+
+ $ sudo -u www-data php occ background:ajax
+   Set mode for background jobs to 'ajax'
+
+The other two commands are:
+
+* ``background:cron``
+* ``background:webcron``
+
+Database Conversion
+-------------------
+
+The SQLite database is good for testing, and for ownCloud servers with small 
+workloads, but production servers with multiple users should use MariaDB, MySQL, 
+or PostgreSQL. You can use ``occ`` to convert from SQLite to one of these other 
+databases. You need:
+
+* Your desired database installed and its PHP connector
+* The login and password of a database admin user
+* The database port number, if it is a non-standard port
+
+This is example converts to SQLite MySQL/MariaDB:: 
+
+ $ sudo -u www-data php occ db:generate-change-script
+ $ sudo -u www-data php occ db:convert-type mysql oc_dbuser 127.0.0.1 
+ oc_database
+
+For a more detailed explanation see 
+:doc:`../configuration_database/db_conversion`
+
+File Operations
+---------------
+
+The ``files:scan`` command scans for new files for the file cache, and isn't 
+intended to be run manually.
+
+``files:cleanup`` tidies up the server's file cache by deleting all file 
+entries that have no matching entries in the storage table.
+   
+   
+l10n, Create javascript Translation Files for Apps
+--------------------------------------------------
+
+Use the ``l10n:createjs`` to translate apps into various languages, using this 
+syntax::
+
+  l10n:createjs appname language_name
+  
+The output can be either ``.js`` or ``.json``. This example converts the 
+Activity app to Bosnian::
+
+ $ sudo -u www-data php occ l10n:createjs activity bs.js
```

Comment:
I need some clarification on this-- I couldn't test it because it's broken (there is a fix somewhere), and each language code is duplicated with both .js and .json files, like bs.js and bs.json